### PR TITLE
Update to OTP-19.0.3

### PR DESCRIPTION
**Pull request by joedevivo**

### CODE REVIEW

**Comment by c0b** [comment]
thanks for the PR; I think #17 can be closed as result of this, could you also make a version bump for OTP-18.3.4.3 ?

https://github.com/erlang/otp/releases

**Comment by joedevivo** [comment]
Sure thing. I'll get it in first thing tomorrow morning. Do you think you'd be able to push the images to docker hub sometime tomorrow?

**Comment by c0b** [comment]
I think again, #17 is better merged, please rebase and have a look there in 81f3be8, it's not only `19/Dockerfile` one file, please also make same changes in `19/slim/Dockerfile`

> Do you think you'd be able to push the images
the official images are not update by push, we don't have push permission at all; but instead need a PR, see docker-library/official-images#1943 as a reference.

**Comment by joedevivo** [comment]
Updated to 19.0.3 and 18.3.4.3

File: 18/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-ENV OTP_VERSION="18.3.4.2"
+ENV OTP_VERSION="18.3.4.3"
 
 # We'll install the build dependencies for erlang-odbc along with the erlang
 # build process:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-$OTP_VERSION.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="9c3443c378115c271d0ce214f404f4f602aeec196040ce4a4e8b2707ac0fd4b0" \
+	&& OTP_DOWNLOAD_SHA256="9cfb0315ae2650c3b1bb6f66f430753593f014698fd6105ef19cfd8eb90f72fe" \"
```
Comment (c0b) [inline 18/Dockerfile]: please have a look the CI build failure log, looks like this line has a superfluous `"`

https://travis-ci.org/c0b/docker-erlang-otp/jobs/151225306#L227

File: 18/slim/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-ENV OTP_VERSION="18.3.4.2"
+ENV OTP_VERSION="18.3.4.3"
 
 # We'll install the build dependencies, and purge them on the last step to make
 # sure our final image contains only what we've just built:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-$OTP_VERSION.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="9c3443c378115c271d0ce214f404f4f602aeec196040ce4a4e8b2707ac0fd4b0" \
+	&& OTP_DOWNLOAD_SHA256="9cfb0315ae2650c3b1bb6f66f430753593f014698fd6105ef19cfd8eb90f72fe" \"
```
Comment (c0b) [inline 18/slim/Dockerfile]: and here

**Comment by joedevivo** [comment]
pushed fixes

**Comment by joedevivo** [comment]
Travis passed :D

**Comment by c0b** [comment]
it's merged and please make a PR to official-images, similar to docker-library/official-images#1943

**Comment by joedevivo** [comment]
Which arguments do I pass to `generate-stackbrew-library.sh`?

**Comment by c0b** [comment]
no arguments; just run it

**Comment by joedevivo** [comment]
```
➜  docker-erlang-otp git:(master) ./generate-stackbrew-library.sh
./generate-stackbrew-library.sh: line 5: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
```

**Comment by c0b** [comment]
your bash might be too old; that `declare -A` is to declare bash associate array, (or hash array or dict in other language), is a feature since bash version 4; you may run it on a recent Linux, or you don't have to use this helper script, if you figured out that `library/erlang` file syntax, it's just plain text and need to update some sha1 commit ids

```console
$                  # Ctrl-x Ctrl-v
GNU bash, version 4.3.42(1)-release (x86_64-pc-linux-gnu)
```

**Comment by joedevivo** [comment]
upgrading my bash did it

**Pull request merged**